pkg/containerrun/containerrun.go

```diff
@@ -347,20 +349,33 @@ func startMainProcess(
 	processRegistry.Register(process)
 
 	sentinel := fmt.Sprintf("/var/vcap/data/%s/%s_containerrun.running", jobName, processName)
-	file, _ := os.Create(sentinel)
-	_ = file.Close()
+	err = ioutil.WriteFile(sentinel, nil, 0755)
```

Comment:
Hard coding to /var/vcap/data here and /var/vcap/sys/run for the PID will cause the tests to fail.  This is because on for example a macbook you do not have those locations or permission in the test run to create those locations.

This should be parameterized so that you default to /var/vcap/sys/run and /var/vcap/data normally but also have the ability (as required in the tests) to override this